CefSharp.MinimalExample.Wpf/CefSharp.MinimalExample.Wpf.csproj

```diff
@@ -67,8 +67,10 @@
     <ApplicationManifest>app.manifest</ApplicationManifest>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.Expression.Interactions, Version=4.5.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL" />
```

Comment:
> I Cloned the original project and it doesn't build, some projects appear to be missing, I didn't read any tutorials on the process of contributing to the main project, so I am sorry if this is explained there somewhere.

You'll need to install the `VC++ Development` tools (from memory they are included by default with `VC++`, maybe someone unchecked them upon install?)

> Adding this behavior into the main project would just need to add the references to these new dlls to the Nuget package, what else should be needed?
> 
> I am not understanding your concern with deployment, maybe I am not experienced enough on deployment. IMHO the dlls will be copied to the output folder and the Nuget Package will include them, the only side effect I can see is increasing the project deployment size

It's quite a bit more complicated than that. Using `MahApps` as an example, if I search their issue tracker e.g. https://github.com/MahApps/MahApps.Metro/search?utf8=%E2%9C%93&q=System.Windows.Interactivity.dll&type=Issues I see there are a number of versioning issues.

> but I can fix this changing the Behavior from a Blend Behavior to an Attached Property Behavior

This maybe the simplest option for now.